keps/sig-network/2610-allports-services/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: AllPorts Services
+kep-number: 2610
+authors:
+  - "@prameshj"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cloud-provider
+status: provisional
+creation-date: 2021-04-08
+reviewers:
+  - "@thockin"
+  - "@aojea"
+  - "@danwinship"
+  - "@uablrek"
+  - "@bowei"
+  - "@freehan"
+approvers:
+  - "@thockin"
+prr-approvers:
+  - "@johnbelamaric"
```

Comment:
please reassign to me - John seem to be overloaded this cycle.

Also - you will need to fill in the necessary PRR questions for alpha (not sure if you want to do that in this PRR or in a once that will move it move implementable.

Reply:
Thanks Wojciech. I have reassigned to you.. This will not be targeting alpha in 1.23, most likely the next release.

Reply:
OK - then I will get back to it after feature freeze.